HyperPlay-Gaming/hyperplay-desktop-client
# [Feature] Browser game close through overlay

Browser games can now be killed through the game card and through the game overlay

### Testing
1. Launch Defi Kingdoms
2. Click X on the game card
3. Game should close

1. Launch Defi Kingdoms
2. Toggle on overlay
3. Click close game button in top right
4. Game should close


---

Use the following Checklist if you have changed something on the Backend or Frontend:
